package.json

```diff
@@ -27,7 +27,7 @@
     "chalk": "^2.1.0",
     "debug": "^3.0.0",
     "generator-ibm-cloud-enablement": "0.8.9",
-    "generator-ibm-service-enablement": "0.9.5",
+    "generator-ibm-service-enablement": "0.9.4",
```

Comment:
This has to be `0.9.3` since it's the prior version with CloudEnvironment at 7.1.0, [here](https://github.com/ibm-developer/generator-ibm-service-enablement/blob/v0.9.3/generators/language-swift-kitura/templates/dependencies.txt)

Reply:
The `dependencies.txt` file in service-enablement will be removed